Firestore/Source/API/FIRFirestore.m

```diff
@@ -50,13 +50,14 @@ @interface FIRFirestore ()
 @property(nonatomic, strong) id<FSTCredentialsProvider> credentialsProvider;
 @property(nonatomic, strong) FSTDispatchQueue *workerDispatchQueue;
 
-@property(nonatomic, strong) FSTFirestoreClient *client;
+@property(nonatomic, strong, readonly) FSTFirestoreClient *client;
```

Comment:
Hrm. Perhaps we've discussed this before, but isn't it disingenuous to mark something readonly if it is modified after the initializer runs?  Perhaps we should at least add a comment to flag the actual behavior?